src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -249,15 +251,54 @@ private string GetResourceStreamName(CultureInfo culture)
             return resourceStreamName;
         }
 
+        private const string AssemblyElementDelimiter = ", ";
+
+        private string ApplyCultureToAssembly(string assemblyFullName, CultureInfo culture)
+        {
+            var assemblyNameElements = assemblyFullName
```

Comment:
You can use this instead
```csharp
private static readonly char[] _assemblyElementSeparator = new[] { ',' , ' ' };
```
after that you can use the array in ```Split``` without creating an array everytime